public/app/features/search/page/components/SearchResultsTable.tsx

```diff
@@ -141,14 +144,42 @@ export const SearchResultsTable = React.memo(
         return (
           <div key={key} {...rowProps} className={className}>
             {row.cells.map((cell: Cell, index: number) => {
+              const href = onClickItem ? url : undefined;
+
+              let userProps: { href?: string; onClick?: (event: React.MouseEvent<HTMLElement>) => void } = {
+                href,
+                onClick: onClickItem,
+              };
+
+              if (cell.column.id === 'column-name' && href) {
+                const parent = response.view.dataFrame.fields[5].values[rowIndex];
+                const kind = response.view.dataFrame.fields[0].values[rowIndex];
+                const existingOnClick = onClickItem;
+
+                userProps.onClick = (evt: React.MouseEvent<HTMLElement>) => {
+                  try {
+                    reportInteraction('grafana_browse_dashboards_page_click_list_item', {
+                      itemKind: kind,
+                      parent: !parent ? 'general' : parent === 'general' ? 'general' : 'folder',
+                      source: trackingSource,
+                    });
+                  } catch (e) {
+                    // ignore analytics errors
+                  }
+                  if (typeof existingOnClick === 'function') {
+                    existingOnClick(evt);
+                  }
```

Comment:
```suggestion
                  if (onClickItem) {
                    onClickItem(evt);
                  }
```
I don't think we need to alias existing onClick.